src/sentry/models/groupowner.py

```diff
@@ -19,6 +24,18 @@ class GroupOwnerType(Enum):
 }
 
 
+class OwnersSerialized(TypedDict):
+    type: GroupOwnerType
+    owner: str
+    dateAdded: models.DateTimeField
```

Comment:
nevermind keep this as date_added. my bad, didn't realize we had already shipped date_added and you're adding types for it